docs/modules/ROOT/pages/databricks.adoc

```diff
@@ -0,0 +1,84 @@
+= DataBricks Support
+:source-highlighter: rouge
+
+//todo can we outsource most of these steps and just call out the aiSSEMBLE-specific stuff?
+There are multiple ways you can deploy a data delivery pipeline.  One way is to leverage the
+https://databricks.com/product/data-lakehouse[Databricks,role=external,window=_blank] managed environment.
+
+== Creating a cluster
+To deploy a Spark job on Azure Databricks we first need to define a cluster.  The steps for creating a cluster that
+is capable of running your pipeline are listed below.  Start by selecting `Compute` on the far left and then select
+`Create Cluster`.
+
+image::azure-new-cluster.png[]
+1.  Name your cluster
+2.  The runtime needs to support Spark 3.0.1 and Scala 2.12 so choose `Runtime: 7.3 LTS`
+3.  Select a Worker Type with enough resources to run your data delivery pipeline.  You can always go back and change
+this setting so there's no harm in starting small and increasing at a later time.
+4.  The Driver Type can be any type of server or the same as the Worker Type.
+5.  Expand the Advanced Options
+6.  In the Spark Config box you can add java options.  If your project uses Krausening to configure your properties
+you can set the following parameters:
+
+    spark.driver.extraJavaOptions
+    -DKRAUSENING_BASE=/dbfs/FileStore/shared_uploads/project-name/krausening/base
+    -DKRAUSENING_EXTENSIONS=/dbfs/FileStore/shared_uploads/project-name/krausening/databricks
+    -DKRAUSENING_PASSWORD=3uQ2j_=wmP5A2q8b
```

Comment:
Should we replace the pwd value to be <put_password_here> ?  I assume this is not a real password, but it looks and feels real so it might be better to place.

Reply:
✅ UPDATED